### ISSUE TITLE
Auto-update result screen

### ISSUE THREAD

**Issue by kristofwillen** [kristofwillen/Pebble-NMBS#5]
The result screen is static currently.  It would be handy to get automatic updates while staying on the result screen.

**Comment by Hipska** [comment]
Even when going back and selecting the station again, the app says it is downloading data, but seems to be using cached data. Don't know if this is BeTrain problem or problem at source API side.

**Comment by kristofwillen** [comment]
This is certainly an API problem.

**Comment by Hipska** [comment]
Too bad, but then the auto-update of the result screens would also have no effect I guess? Do you have this 'caching' behaviour also?

**Comment by kristofwillen** [comment]
Indeed, all depends on the API behaviour.  Auto-updating the screen in this case wouldn't be of much use then.  

I don't suffer from caching behaviour.  What's your mobile OS ? Android/iOS ?
Your caching problem, is it appearing every time you use BeTrain  ?

**Comment by Hipska** [comment]
iPhone 6 iOS 9.1

I will do some more testing this week.
